types/jquery/index.d.ts

```diff
@@ -25,14 +25,14 @@
 // TypeScript Version: 2.3
 
 declare module 'jquery' {
-    function factory(window: Window): JQueryStatic;
+    function factory(window: Window): JQueryStatic & JQuery;
```

Comment:
This was supposed to handle https://github.com/DefinitelyTyped/DefinitelyTyped/issues/17298. I was getting correct behavior while testing although now I can't reproduce it. I'm looking into it.